### PR TITLE
refactor(global): updated course thumbnails using cloudinary images in sample data

### LINKED ISSUE #37: [FEAT]:  Frontend Setup
### Summary
Setup NextJS with TailwindCSS, ShadCN, ESLint, Prettier,  Husky with commitlint, commitizen, lint staged

### Answer the following questions.
1. Is your feature related to UI Development? No
2. Is your feature related to API Integration? No
3. Is your feature related to repository setup? Yes

### Resources

### PR DESCRIPTION
**Pull request by sub1120**
### Describe your changes

- updated next.js.config to allow cloudinary cdn
- updated course thumbnail images in sample data.
- updated types names for course 

### Issue ticket number and link

Write here...

### Please confirm the below message,

- [X] I confirm that I have performed a self-review of my code, and written explainable comments in the code, the UI is rendered without any errors, the UI is responsive, dark mode UI is implemented, removed all debug-related code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,29 +9,41 @@
...
 pnpm run dev
 ```
 
 ## Project Structure
+
 1. Code files
+
 - `src/`: Contains main source code
 - `src/app`: Next.js app directory. [Checkout Next.js Routing](https://nextjs.org/docs/app/building-your-application/routing)
 - `src/app/global.css`: Global CSS file.
```
Comment (shivamvijaywargi) [inline README.md]: globals.css

File: README.md [hunk 2]
```diff
@@ -58,6 +72,7 @@ pnpm run dev
 - `package.json`: Project dependencies.
```
Comment (shivamvijaywargi) [inline README.md:72]: tailwind config?

**Review by shivamvijaywargi — CHANGES_REQUESTED** [review]
Please check the comments

File: README.md (hunk 2 above)
Comment (sub1120) [reply README.md:72]: It is present bro.

File: README.md (hunk 1 above)
Comment (sub1120) [reply README.md]: Corrected.

**Review by shivamvijaywargi — APPROVED** [review]

**Pull request merged**